cli/cli-client/src/main/resources/help/shutdown.adoc

```diff
@@ -0,0 +1,20 @@
+SHUTDOWN(1)
+===========
+:doctype: manpage
+
+
+NAME
+----
+shutdown - shuts down a single server or the whole cluster
+
+
+SYNOPSIS
+--------
+*shutdown server* ['SERVER']
+
+*shutdown cluster*
+
+
+DESCRIPTION
+-----------
+The *shutdown* command performs an orderly shutdown of a single server node or of the whole cluster.
```

Comment:
There are docs for the CLI, but not for the REST endpoint

Reply:
Right